.github/workflows/publish_image_chart.yaml

```
@@ -147,14 +147,31 @@ jobs:
           labels: helmRepo=pipecd
           data: ${{ env.PIPECD_VERSION }}
 
+  update-quickstart:
```

```suggestion
  release-quickstart-manifests:
```
How about rename like this?

---

I think its better. Thank you.